lib/server.js

```diff
@@ -214,12 +219,14 @@ function setupConfigDefaults(conf) {
 }
 
 // Main app setup
-function main(conf) {
-    conf = setupConfigDefaults(conf);
+function main(options) {
+    var conf = setupConfigDefaults(options.config);
     // Set up the global options object with a logger
     var opts = {
-        log: rbUtil.makeLogger(conf.logging),
-        conf: conf
+        conf: conf,
+        logger: options.logger,
+        log: options.logger.log.bind(options.logger),
```

Comment:
The service runner code no longer sets up the magic function that also has a .child method attached to it. Instead, it exposes a plain logger object.

I'm torn about re-introducing the magic. It sure is more convenient for users, just feels a bit dirty.